index.html

```diff
@@ -2,45 +2,39 @@
 <html lang="en">
 	<head>
 		<meta charset="utf-8" />
-		<meta name="viewport" content="width=device-width, initial-scale=1" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
 		<title>African Development Global Forum</title>
-		<link
-			href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css"
-			rel="stylesheet"
-			integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi"
-			crossorigin="anonymous"
-		/>
 		<link
 			href="./node_modules/@fortawesome/fontawesome-free/css/all.css"
 			rel="stylesheet"
 		/>
-
-		<link
-			rel="stylesheet"
-			href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
-		/>
 		<link rel="stylesheet" href="./assets/css/style.css" />
 	</head>
 	<body>
 		<div class="wraper">
 			<header>
 				<nav class="menu-container">
-					<div class="menu">
+					<button class="menu-toggler" type="button">
+							<span class="menu-toggler-icon"><i class="fa fa-bars"></i></span>
+						</button>
+                        
+                        <div class="menu">
 						<a class="menu-brand" href="index.html"
 							><img src="./assets/images/logo.png" width="65px" alt="ADF logo"
 						/></a>
-						<button class="menu-toggler" type="button">
-							<span class="menu-toggler-icon"><i class="fa fa-bars"></i></span>
-						</button>
+						
 						<div class="menu-collapse" id="navbarNavAltMarkup">
 							<div class="menu-nav">
-								<a class="nav-link active" aria-current="page" href="about.html"
+								<a class="nav-link active" aria-current="page" href="index.html"
+									>Home</a
+								>
+								<a class="nav-link" aria-current="page" href="about.html"
 									>About</a
 								>
 								<a class="nav-link" href="#">Program</a>
 								<a class="nav-link" href="#">Join</a>
 								<a class="nav-link" href="#">Sponsor</a>
-								<a class="nav-link" href="#">News</a>
+
 								<a class="nav-link btn">ADF Campaign</a>
```

Comment:
- [ ] There are two navbars in the original design, please 🙏🏽 add the black navbar 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/197135584-1536ddf1-2e92-4678-97c3-800e2e76cc32.png)